tests/test_conversationhandler.py

```diff
@@ -753,6 +753,49 @@ def test_all_update_types(self, dp, bot, user1):
         assert not handler.check_update(Update(0, pre_checkout_query=pre_checkout_query))
         assert not handler.check_update(Update(0, shipping_query=shipping_query))
 
+    def test_promise_exception(self, dp, bot, user1, caplog):
+        def conv_entry(*a, **kw):
+            return 1
+
+        def raise_error(*a, **kw):
+            raise Exception("Oh no")
```

Comment:
Please try to use some more descriptive strings ;)

Reply:
Okay 😅